src/components/submitComment/index.js

```diff
@@ -1,77 +1,116 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './styles.css';
 
-const SubmitComment = ({
-  onUserCommentChange,
-  onCommentSubmit,
-  userComment,
-  title = 'Add a comment:',
-  type = 'Add',
-  onCommentCancel,
-  comment = {},
-}) => {
-  const { _id = null, userEmail = '', userName = '' } = comment;
+class SubmitCommentForm extends Component {
+  constructor(props) {
+    super(props);
 
-  const formTitle =
-    type === 'Edit' ?
-      <h2>{title}: <span>{`${userName} - ${userEmail}`}</span></h2>
-      :
-      <h2>{title}</h2>
-    ;
+    this.state = {
+      hasErrors: false
+    };
 
-  return (
-    <section className="blabbr-submitComment">
-      <form>
-        {formTitle}
-        <textarea
-          value={userComment}
-          onChange={onUserCommentChange}
-        />
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
 
-        { type === 'Edit' ?
-        [
+  handleChange(event) {
```

Comment:
handleChange = (event) => {

then you don't need to bind it in constructor. cleaner.